demo/demo.pro

```diff
@@ -1,31 +1,44 @@
-ADS_ROOT = $${PWD}/..
 ADS_OUT_ROOT = $${OUT_PWD}/..
 
 TARGET = AdvancedDockingSystemDemo
 DESTDIR = $${ADS_OUT_ROOT}/lib
 QT += core gui widgets
-CONFIG *= c++14
+CONFIG += c++11
```

Comment:
What is the reason for releaxing the required C++ version from c++14 to c++11?

Reply:
more compatibility with compilers. the library doesn't really use anything that C++14 does better than C++11

Reply:
Yes, this is right. But it also prevents users in future contributions from using C++14 features. But I will accept this change for now.

Reply:
That's a fair point. I know how to make C++14 recommended but not required using CMake but I don't think there's a similar concept in qmake

Reply:
I would say, if there are no problems with any compiler that is tested by Travis, we should stay with C++14 and revert this change. If there is a problem, we switch to C++11.